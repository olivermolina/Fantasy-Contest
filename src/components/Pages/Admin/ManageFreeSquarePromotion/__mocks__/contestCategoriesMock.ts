import { ContestCategory, Prisma } from '@prisma/client';
import { faker } from '@faker-js/faker';

export const contestCategoriesMock: ContestCategory[] = [
  {
    id: faker.datatype.uuid(),
    numberOfPicks: 2,
    allInPayoutMultiplier: 3,
    primaryInsuredPayoutMultiplier: 2,
    secondaryInsuredPayoutMultiplier: 0.5,
    customStakeLimitEnabled: false,
    maxStakeAmount: new Prisma.Decimal(0),
    minStakeAmount: new Prisma.Decimal(0),
    bonusCreditLimitId: null,
  },
  {
    id: faker.datatype.uuid(),
    numberOfPicks: 3,
    allInPayoutMultiplier: 5,
    primaryInsuredPayoutMultiplier: 2.5,
    secondaryInsuredPayoutMultiplier: 1.25,
    customStakeLimitEnabled: false,
    maxStakeAmount: new Prisma.Decimal(0),
    minStakeAmount: new Prisma.Decimal(0),
    bonusCreditLimitId: null,
  },
  {
    id: faker.datatype.uuid(),
    numberOfPicks: 4,
    allInPayoutMultiplier: 10,
    primaryInsuredPayoutMultiplier: 5,
    secondaryInsuredPayoutMultiplier: 1.25,
    customStakeLimitEnabled: false,
    maxStakeAmount: new Prisma.Decimal(0),
    minStakeAmount: new Prisma.Decimal(0),
    bonusCreditLimitId: null,
  },
];
