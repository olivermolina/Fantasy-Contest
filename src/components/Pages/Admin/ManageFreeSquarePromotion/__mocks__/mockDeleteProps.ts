export const mockDeleteProps = {
  open: true,
  setOpen: () => console.log('open'),
  freeSquareId: '123',
  handleDelete: () => console.log('delete'),
};
