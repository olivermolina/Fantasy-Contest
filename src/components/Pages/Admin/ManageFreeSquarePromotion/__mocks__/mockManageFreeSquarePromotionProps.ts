import { League } from '@prisma/client';
import { contestCategoriesMock } from './contestCategoriesMock';

export const mockManageFreeSquarePromotionProps = {
  data: [
    {
      id: '63182-83-5227',
      selId: 5227,
      statName: 'Points',
      total: 11.5,
      matchName: 'STAN @ ORE ST',
      playerName: 'Glenn Taylor Jr.',
      marketId: '63182-83-5227',
      league: 'NCAAB',
      matchTime: 'Mar 02, 2023 @ 09:00 PM',
      freeSquare: {
        id: '1',
        discount: 95,
        pickCategories: [
          {
            id: '2 pick',
            numberOfPicks: 2,
          },
          {
            id: '3 pick',
            numberOfPicks: 3,
          },
          {
            id: '4 pick',
            numberOfPicks: 4,
          },
        ],
        maxStake: 25,
        freeEntryEnabled: false,
      },
      freeSquareDiscount: 95,
      discountedTotal: 1,
      picksCategory: '2,3,4',
      maxStake: 25,
      freeEntryEnabled: false,
    },
    {
      id: '63194-86-3541',
      selId: 3541,
      statName: 'Points Rebounds and Assists',
      total: 16.5,
      matchName: 'ARIZ @ USC',
      playerName: 'Kerr Kriisa',
      league: 'NCAAB',
      matchTime: 'Mar 02, 2023 @ 11:00 PM',
      freeSquare: {
        id: '1',
        discount: 98,
        pickCategories: [
          {
            id: '2 pick',
            numberOfPicks: 2,
          },
          {
            id: '3 pick',
            numberOfPicks: 3,
          },
          {
            id: '4 pick',
            numberOfPicks: 4,
          },
        ],
        maxStake: 25,
        freeEntryEnabled: false,
      },
      freeSquareDiscount: 95,
      discountedTotal: 1,
      picksCategory: '2,3,4',
      maxStake: 25,
      freeEntryEnabled: false,
    },
  ],
  setSelectedLeague: () => alert('setSelectedLeague'),
  handleSave: () => console.log('save'),
  contestCategories: contestCategoriesMock,
  selectedLeague: League.NBA,
  handleDelete: () => console.log('delete'),
};
