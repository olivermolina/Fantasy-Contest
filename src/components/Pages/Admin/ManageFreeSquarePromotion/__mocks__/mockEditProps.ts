import { mockManageFreeSquarePromotionProps } from './mockManageFreeSquarePromotionProps';
import { contestCategoriesMock } from './contestCategoriesMock';

export const mockEditProps = {
  row: mockManageFreeSquarePromotionProps.data[0],
  open: true,
  handleClose: () => console.log('close'),
  contestCategories: contestCategoriesMock,
  handleSave: () => console.log('save'),
};
