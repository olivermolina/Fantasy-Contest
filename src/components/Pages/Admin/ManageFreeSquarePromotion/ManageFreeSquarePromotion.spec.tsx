import { render } from '@testing-library/react';
import ManageFreeSquarePromotion from './ManageFreeSquarePromotion';
import { mockManageFreeSquarePromotionProps } from './__mocks__/mockManageFreeSquarePromotionProps';

describe('ManageFreeSquarePromotion', () => {
  it('should render correctly', () => {
    const { container } = render(
      <ManageFreeSquarePromotion {...mockManageFreeSquarePromotionProps} />,
    );
    expect(container).toMatchSnapshot();
  });
});
