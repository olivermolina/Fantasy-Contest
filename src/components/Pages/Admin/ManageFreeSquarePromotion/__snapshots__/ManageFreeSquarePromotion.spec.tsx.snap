// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ManageFreeSquarePromotion should render correctly 1`] = `
<div>
  <div
    class="flex flex-col gap-2 p-2"
  >
    <div
      class="flex flex-row gap-2 items-center justify-between"
    >
      <div
        class="MuiFormControl-root MuiTextField-root css-10qkx7n-MuiFormControl-root-MuiTextField-root"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiFormLabel-filled MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined css-1sumxir-MuiFormLabel-root-MuiInputLabel-root"
          data-shrink="true"
          for="league"
          id="league-label"
        >
          Select league
        </label>
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
        >
          <div
            aria-expanded="false"
            aria-haspopup="listbox"
            aria-labelledby="league-label league"
            class="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall css-jedpe8-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
            id="league"
            role="button"
            tabindex="0"
          >
            NBA
          </div>
          <input
            aria-hidden="true"
            class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
            tabindex="-1"
            value="NBA"
          />
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-hfutr2-MuiSvgIcon-root-MuiSelect-icon"
            data-testid="ArrowDropDownIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M7 10l5 5 5-5z"
            />
          </svg>
          <fieldset
            aria-hidden="true"
            class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="css-14lo706"
            >
              <span>
                Select league
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        class="max-w-lg"
      >
        <div
          class="relative grow"
        >
          <div
            class="flex absolute inset-y-0 left-0 items-center pl-3 pointer-events-none"
          >
            <svg
              aria-hidden="true"
              class="w-5 h-5 text-gray-500 dark:text-gray-400"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
              />
            </svg>
          </div>
          <input
            class="block p-4 pl-10 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-2 focus:ring-blue-500 focus:border-blue-500 focus:outline-none"
            placeholder="Search"
            required=""
            type="search"
            value=""
          />
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root css-8atqhb"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1mxz8qt-MuiPaper-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular bg-blue-500 text-white css-xvd0n5-MuiToolbar-root"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
          >
            Free Square Promotions
          </h5>
        </div>
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root css-11xur9t-MuiPaper-root-MuiTableContainer-root"
        >
          <table
            aria-label="Free Square Promotions"
            class="MuiTable-root css-rqglhn-MuiTable-root"
          >
            <thead
              class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
            >
              <tr
                class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      ID
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      League
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Player name
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Stat name
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Match DateTime
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Match Up
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Total Stat
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Free Square Discount
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Discounted Total
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Max Stake/Bet
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      # of Players
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class="cursor-pointer select-none"
                    >
                      Can use bonus credit (Free Entry)
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignCenter MuiTableCell-sizeMedium css-1uhqakm-MuiTableCell-root"
                  colspan="1"
                  scope="col"
                >
                  <span
                    class="MuiButtonBase-root MuiTableSortLabel-root css-1qgma8u-MuiButtonBase-root-MuiTableSortLabel-root"
                    role="button"
                    tabindex="0"
                  >
                    <div
                      class=""
                    >
                      Action
                    </div>
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                      data-testid="ArrowDownwardIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                      />
                    </svg>
                  </span>
                </th>
              </tr>
            </thead>
            <tbody
              class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
            >
              <tr
                class="MuiTableRow-root MuiTableRow-hover css-1q1u3t4-MuiTableRow-root"
              >
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  <div
                    class="css-7owdng-MuiStack-root"
                  >
                    <div
                      class="MuiBox-root css-182y0me"
                    >
                      63182-83-5227
                    </div>
                    <button
                      aria-label="Copied to clipboard!"
                      class=""
                      data-mui-internal-clone-element="true"
                    >
                      <svg
                        class="h-6 w-6"
                        data-file-name="SvgClipboardDocument"
                      />
                    </button>
                  </div>
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  NCAAB
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  Glenn Taylor Jr.
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  Points
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  Mar 02, 2023 @ 09:00 PM
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  STAN @ ORE ST
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  11.5
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  95
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  1.00
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  25.00
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  2,3,4
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                />
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  <div
                    class="flex justify-around gap-1"
                  >
                    <button
                      class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeSmall MuiButton-outlinedSizeSmall MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeSmall MuiButton-outlinedSizeSmall css-1k23hlb-MuiButtonBase-root-MuiButton-root"
                      tabindex="0"
                      type="button"
                    >
                      Edit
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                    <button
                      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall MuiButton-containedSizeSmall MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall MuiButton-containedSizeSmall css-11qr2p8-MuiButtonBase-root-MuiButton-root"
                      tabindex="0"
                      type="button"
                    >
                      Remove
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </td>
              </tr>
              <tr
                class="MuiTableRow-root MuiTableRow-hover css-1q1u3t4-MuiTableRow-root"
              >
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  <div
                    class="css-7owdng-MuiStack-root"
                  >
                    <div
                      class="MuiBox-root css-182y0me"
                    >
                      63194-86-3541
                    </div>
                    <button
                      aria-label="Copied to clipboard!"
                      class=""
                      data-mui-internal-clone-element="true"
                    >
                      <svg
                        class="h-6 w-6"
                        data-file-name="SvgClipboardDocument"
                      />
                    </button>
                  </div>
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  NCAAB
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  Kerr Kriisa
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  Points Rebounds and Assists
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  Mar 02, 2023 @ 11:00 PM
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  ARIZ @ USC
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  16.5
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  95
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  1.00
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  25.00
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  2,3,4
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                />
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignCenter MuiTableCell-sizeMedium css-3ybl5n-MuiTableCell-root"
                >
                  <div
                    class="flex justify-around gap-1"
                  >
                    <button
                      class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeSmall MuiButton-outlinedSizeSmall MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeSmall MuiButton-outlinedSizeSmall css-1k23hlb-MuiButtonBase-root-MuiButton-root"
                      tabindex="0"
                      type="button"
                    >
                      Edit
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                    <button
                      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall MuiButton-containedSizeSmall MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeSmall MuiButton-containedSizeSmall css-11qr2p8-MuiButtonBase-root-MuiButton-root"
                      tabindex="0"
                      type="button"
                    >
                      Remove
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>
</div>
`;
